README.rst

```diff
@@ -94,9 +94,10 @@ Using a config file
 Command line options can also be specified in a config file.
 
 When running ``codespell``, it will check in the current directory for a file
-named ``setup.cfg`` or ``.codespellrc`` (or a file specified via ``--config``),
-containing an entry named ``[codespell]``. Each command line argument can
-be specified in this file (without the preceding dashes), for example::
+named ``setup.cfg``, ``tox.ini`` or ``.codespellrc`` (or a file specified via
```

Comment:
I think the order in the readme should match the order they are loaded in the code:
```suggestion
named ``setup.cfg``, ``.codespellrc`` or ``tox.ini`` (or a file specified via
```

Reply:
Fixed the order of configuration files in the documentation.

Reply:
The priority of configuration files is an issue in theory, but is it worth addressing in practice? Because `tox.ini` was not a configuration file (until now) what are the chances that existing `tox.ini` files in the wild contain a `[codespell]` section?

Additionally, the documentation of [`configparser.ConfigParser.read()`](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read) does not specify how it behaves when passed an iterable of potential configuration file locations. Which file has priority? My guess is that the latest file actually has priority, as its options will overwrite identical options read from previous files. That's only a guess of course, but if so, `tox.ini` should be added first instead of last to have the lowest priority. As an alternative, we could read each possible configuration file separately and merge them manually in a well-defined order. But then, I feel we shouldn't address such details if even the [`configparser`](https://docs.python.org/3/library/configparser.html) modules does not.

Reply:
I have actually modified my commit to use the reverse order from [configparser.ConfigParser.read()](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read) in the documentation.

Reply:
I had a look at [`configparser.RawConfigParser._read()`](https://github.com/python/cpython/blob/53a54b781d1f05f2d0b40ce88b3da92d5d23e9d2/Lib/configparser.py#L973) and indeed the last file takes precedence over previous files.

I ended up opening Python bug  https://github.com/python/cpython/issues/96765.